backend/cmd/zeevision/main.go

```diff
@@ -6,10 +6,30 @@ import (
 	"github.com/ducanhpham0312/zeevision/backend/internal/consumer"
 	"github.com/ducanhpham0312/zeevision/backend/internal/endpoint"
 	"github.com/ducanhpham0312/zeevision/backend/internal/environment"
+	"github.com/ducanhpham0312/zeevision/backend/internal/storage"
 )
 
 // Entry point for the application.
 func main() {
+
+	dsnConfig := storage.DsnConfig{
+		User:         "zeevision_user",
+		Password:     "zeevision_pass",
```

Comment:
Remember to change `docker-compose.yml` to set these environment variables!

Reply:
Can you explain more specifically ?

Reply:
Set User and Password from environment variables instead of hard-coding them in. Don't give User nor Password any default values like other variables in environment.go have.